proto/regen/ecocredit/basket/v1alpha1/types.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package regen.ecocredit.basket.v1alpha1;
+
+option go_package = "github.com/regen-network/regen-ledger/x/ecocredit/basket/v1alpha1";
+
+// BasketCredit represents the information for a credit batch inside a basket.
+message BasketCredit {
+
+  // batch_denom is the unique ID of the credit batch.
+  string batch_denom = 1;
+
+  // tradable_amount is the number of credits in this transfer that can be
+  // traded by the recipient. Decimal values are acceptable within the
+  // precision returned by Query/Precision.
+  string tradable_amount = 2;
```

Comment:
```suggestion
  // amount is the number of credits associated with the given `batch_denom`
  // Decimal values are acceptable within the precision of the corresponding
  // credit type for this batch.
  string amount = 2;
```

Suggesting changing `tradable_amount` => `amount` as these credits may be retired when sent back to the user if `retire_on_take` is true.